config/initializers/blacklight_initializer.rb

```diff
@@ -5,3 +5,5 @@
 
 # Blacklight.secret_key = '92188d94fcaa0d8558b6e39d71a07e17a51ac952aced31d1a116d8aab4cd5f1cf862fde4bd0242865a7be1b7a32d3e2b2d232602969d9d2a6bd132edfe445833'
 
+# Increase Blacklight config max results per page
+Blacklight::Configuration.default_values[:max_per_page] = 10000
```

Comment:
Did you already look into this? https://github.com/projectblacklight/blacklight/wiki/Pagination

Reply:
Other pages in the repo use pagination but I don't think it would work in this case because this list of results are used in a form. There's a page refresh each time you select a different page of results which would make you lose any entries you'd selected.

Also we don't want to invest a lot of time fixing this since we're planning to upgrade to hyrax soon anyway.